tests/test_zookeeper.py

```diff
@@ -183,8 +190,12 @@ def test_update_leader(self):
     def test_write_leader_optime(self):
         self.zk.last_leader_operation = '0'
         self.zk.write_leader_optime('1')
+        with patch.object(MockKazooClient, 'create_async', Mock()):
```

Comment:
create_async is defined as create or Mock, what's the purpose of overriding it here with Mock() ?